src/nodes.coffee

```diff
@@ -1501,6 +1505,15 @@ exports.Obj = class Obj extends Base
           [key, value] = prop.base.cache o
           key  = new PropertyName key.value if key instanceof IdentifierLiteral
           prop = new Assign key, value, 'object'
+        else if key instanceof Value and key.base instanceof ComputedPropertyName
+          # { [foo()] } => { [ref = foo()]: ref }
```

Comment:
Do you mean:

```
# `{ [foo()] }` output as `{ [ref = foo()]: ref }`.
```
? Remember the comments are markdown and become the annotated source.

Reply:
Right. I'll keep that in mind.